### PR TITLE
Adding support of multi directives

### PR DESCRIPTION
**Pull request by MorganDbs**
Hello, 

We really enjoyed the lib but multi directives support was missing and we so the "TODO", so here it is :)

We also added test for this use case.

Hope this will be ok for you and make it available to everyone

### CODE REVIEW

**Review by vkolgi — CHANGES_REQUESTED** [review]
Thanks for submitting the PR to improve the repo. I have a minor suggestion, can you take a look.

File: src/jsonToGraphQLQuery.ts [hunk 1]
```diff
@@ -115,14 +115,14 @@ function convertQuery(node: any, level: number, output: [string, number][], opti
...
-                                `but only 1 directive per object/key is supported at this time.`);
-                        }
-                        dirsStr = `@${buildDirectives(value.__directives)}`;
+                        Object.keys(value.__directives).forEach(
+                            (directive, index) => {
+                                dirsStr += `@${buildDirectives({
+                                    [directive]: value.__directives[directive],
+                                })}${index < numDirectives - 1 ? ' ' : ''}`;
+                            }
+                        );
```
[the remark this answers is not in the document]
Comment (MorganDbs) [reply src/jsonToGraphQLQuery.ts]: Yes you are right, your's is better !
I just pushed the changes :)

[the remark this answers is not in the document]
Comment (vkolgi) [reply src/jsonToGraphQLQuery.ts]: @MorganDbs One issue with my suggestion was that typescript complains of no whitespace between the colons. This is trivial but fails the npm run build. Can you please add a space between the colon here and commit?. `{[item[0]]:item[1]})}`. 
You can check if this works in local by running `npm run build`.

[the remark this answers is not in the document]
Comment (MorganDbs) [reply src/jsonToGraphQLQuery.ts]: @vkolgi Done :)

**Review by vkolgi — APPROVED** [review]
The changes looks good.

**Comment by MorganDbs** [comment]
> The changes looks good.

@vkolgi 

Thanks to you ! 
Any idea on when you'll release the new version ?

**Comment by vkolgi** [comment]
@MorganDbs I will bump the minor version and release today.

**Comment by MorganDbs** [comment]
@vkolgi that's quick ! Thanks a lot

**Pull request merged**